docs/docs/overview/guides/aws.mdx

```diff
@@ -0,0 +1,241 @@
+import {
+RIG_OPERATOR_CHART_VERSION
+} from "../../../src/constants/versions";
+import {
+RIG_PLATFORM_CHART_VERSION
+} from "../../../src/constants/versions";
+import CodeBlock from '@theme/CodeBlock';
+import ThemedImage from "@theme/ThemedImage";
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+# Setting up Rig on EKS on Amazon Web Services
+
+## Prerequisites
+This guide assumes you have a running [EKS](https://aws.amazon.com/eks/) cluster with the [AWS Load Balancer Controller](https://kubernetes-sigs.github.io/aws-load-balancer-controller/v2.8/) installed. [This guide](https://docs.aws.amazon.com/eks/latest/userguide/quickstart.html) by AWS shows how to setup a simple cluster and sample application with the Load Balancer Controller installed as well.
+
+Besides just having an EKS cluster up and running, your `kubectl` should have its current context pointing to your cluster. [Here](https://docs.aws.amazon.com/eks/latest/userguide/create-kubeconfig.html) you can see how to get that.
+
+## Installing the Rig Operator
+
+We install the Rig Operator using Helm. We need one piece of AWS specific configuration to make Ingress work such that your Capsules and Rig Platform is reachable on the Internet. AWS EKS has their own Ingress controller, the Amazon Load Balancer (ALB) Controller mentioned in *Prerequisites*. The Rig Operator needs to be instructed how it generates Ingress resources such that they fit with the ALB. This we do through our [Routes plugin](/operator-manual/setup-guide/operator/networking). The following Helm values configure it to work with the ALB.
+
+```yaml title="Helm values - Operator"
+config:
+  pipeline:
+    routesStep:
+      plugin: rigdev.ingress_routes
+      config: |
+        ingressClassName: alb
+        disableTLS: true
```

Comment:
Add a note about TLS, that it's needed because ALB handles TLS outside of the cluster.